contact.html

```diff
@@ -0,0 +1,51 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Sprint Challenge - Contact</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+</head>
+<body>
+        <div class="container contact-page">
+                <header>
+                    <div class="nav-container">
+                        <div class="logo">
+                            <img src="img/lambda-black.png" alt="Logo">
+                        </div>
+                        <nav>
+                            <a href="index.html">Home</a>
+                            <a href="about.html">About</a>
+                            <a href="#">Products</a>
+                            <a href="#">Blog</a>
+                            <a href="#">Contact</a>
+                        </nav>
+                    </div>
+                <section class="form">
+                    <h2>Kindly Drop Your message</h2>
+                    <form >
+                        <label for="name">Your name:</label>
+                        <input type="text">
+                        <label for="email">Your email:</label> 
+                        <input type="email"> 
+                        <label for=""> Your Message:</label>
+                        <textarea name="" id=""> </textarea>
+                        <button>Submit</button>
+                    </form>
+                </section>
+            <footer>
```

Comment:
I tested your footer on 2 different browser and it looks like it's not placed correctly (it has a huge withe space below).
You probably could play with the height of your container or with a top margin or top padding of your footer. 
This could be a nice time to use your contact-page class for specificity.